SharedSources/Coordinators/AppCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppCoordinator.swift
+//  VLC-iOS
+//
+//  Created by Carola Nitz on 11/30/17.
+//  Copyright © 2017 VideoLAN. All rights reserved.
+//
```

Comment:
Please add " // Refer to the COPYING file of the official project for license." to the header of all your new files to achieve parity with the existing code base.